packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -86,11 +86,6 @@ export class WaferMap extends FoundationElement {
      */
     public renderQueued = false;
 
```

Comment:
I'm missing some context on the goal of this PR. The matrix story is now rendering with its contents outside the bounds. What's the benefit of allowing that?

![image](https://user-images.githubusercontent.com/10500124/217661399-15a290c6-bc3c-48f1-af47-8b8e1ad192f8.png)

Reply:
Our use case does not cover square shaped data, so the fact that previously it was inside bounds was a fortunate bug. I will change the matrix sets to circle shaped data